subsys/net/ip/Kconfig

```diff
@@ -842,6 +842,13 @@ module-help = Enables promiscuous mode to output debug messages.
 source "subsys/net/Kconfig.template.log_config.net"
 endif # NET_PROMISCUOUS_MODE
 
+config NET_STEALTH_UNREACHABLE_DESTINATION
```

Comment:
Please include `ICMP` in the option name. Anyway, `STEALTH` is a bit unclear to me, perhaps it could be simply `NET_DISABLE_ICMP_DESTINATION_UNREACHABLE`.

Reply:
Done.